src/types.rs

```diff
@@ -1120,6 +1246,36 @@ pub struct Policy<Attributes> {
     pub items:             Attributes,
 }
 
+#[cfg(feature = "std")]
+impl quickcheck::Arbitrary for OwnedPolicy {
+    //todo it is not clear to me if random instances of these are useful in tests
+    fn arbitrary(g: &mut Gen) -> OwnedPolicy {
+        OwnedPolicy {
+            identity_provider: quickcheck::Arbitrary::arbitrary(g),
+            created_at:        quickcheck::Arbitrary::arbitrary(g),
+            valid_to:          quickcheck::Arbitrary::arbitrary(g),
+            items:             quickcheck::Arbitrary::arbitrary(g),
```

Comment:
We should not generate arbitrary vectors here. Attribute values can only have limited length. If I recall correctly it's 31 bytes (the definitive value is in rust-src/id/src/constants.rs in concordium-base).

Reply:
Done